test/extended/testdata/bindata.go

```diff
@@ -22044,7 +22061,8 @@ var _examplesQuickstartsNodejsMongodbPersistentJson = []byte(`{
   },
   "message": "The following service(s) have been created in your project: ${NAME}, ${DATABASE_SERVICE_NAME}.\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/openshift/nodejs-ex/blob/master/README.md.",
   "labels": {
-    "template": "nodejs-mongo-persistent"
+    "template": "nodejs-mongo-persistent",
+    "app": "nodejs-mongo-persistent"
```

Comment:
The `app` label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.